base/Dockerfile

```diff
@@ -0,0 +1,42 @@
+#
+# Copyright 2017 Confluent Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM azul/zulu-openjdk-alpine:8
+
+# Remember where we came from
+LABEL io.confluent.docker.git.repo="confluentinc/common-docker"
+
+ARG COMMIT_ID=unknown
+LABEL io.confluent.docker.git.id=$COMMIT_ID
+
+ARG BUILD_NUMBER=-1
+LABEL io.confluent.docker.build.number=$BUILD_NUMBER
+
+MAINTAINER tools@confluent.io
+LABEL io.confluent.docker=true
+
+# This affects how strings in Java class files are interpreted.  We want UTF-8 and this is the only locale in the
+# base image that supports it
+ENV LANG="C.UTF-8"
+
+COPY requirements.txt .
+
+RUN apk update \
+    && apk add bash curl git wget netcat-openbsd python2 py2-pip \
```

Comment:
Do we really need both `curl` and `wget`? Is `git` for cloning our repos or just for installing the git requirement in `requirements.txt`? If the latter, can we just release it instead?

Reply:
This is copied from the original Dockerfile in cp-docker-images, which included both, so I assume they're needed somewhere.

I feel like our own specialized build scripts don't belong in the public pypi. Since git and pip end up getting removed from the final images, I think it should be okay.

Reply:
ack, unfortunate that we ended up including both already :(